cypress/elements/calculationsModal.js

```diff
@@ -0,0 +1,13 @@
+const modalTitleEl = 'calculation-modal-title'
```

Comment:
For consistency in this file, just use the selector directly in the function (line 13) rather than creating a const.

Reply:
done